Dockerfile

```diff
@@ -9,7 +9,8 @@ FROM bellsoft/liberica-openjdk-alpine:11.0.22-12 as arm64
 
 FROM ${TARGETARCH}
 
-ARG SBT_VERSION
+# Add a default version if unset.
+ARG SBT_VERSION=1.9.8
```

Comment:
This was throwing errors when I was using the dockerfile instead of the ubuntu image. It looks like I actually don't need this anymore. Removed and tested and it still works as expected